src/app/overview/overview.component.html

```diff
@@ -10,7 +10,13 @@ <h2 class="col-12" style="margin-top: 15px">
       Food
     </h2>
     <div class="col-12">
-        <mat-progress-bar mode="determinate" value="60"></mat-progress-bar>
+      <div style="display: flex; justify-content: center">
+        <button *ngIf="!counts.total; else count" mat-button color="accent" routerLink="/food">Add some food!</button>
+        <ng-template #count>
```

Comment:
I wonder if this count would look better underneath the progress bar and all the way to the right similar to the mat field hint

Reply:
To be fair I haven't actually seen the current version